response_operations_ui/templates/account/change-password.html

```diff
@@ -80,20 +78,32 @@ <h1 class="ons-u-fs-xl">Change password</h1>
             <h2 class="ons-u-mt-l">Create a new password</h2>
             <div class="ons-message">
                 <p>Your password must have:</p>
-                <ul>
-                    <li>at least 8 characters </li>
-                    <li>at least 1 uppercase letter</li>
-                    <li>at least 1 symbol (eg: ?!£%) </li>
-                    <li>at least 1 number</li>
-                </ul>
+                {{
```

Comment:
Its a question and not a comment. I would have gone with the html tags where we can to avoid any breaking change. Is my thought wrong. please guide.